src/App.tsx

```diff
@@ -17,18 +39,46 @@ export const App: React.FC = () => {
             <h1 className="title">Todos:</h1>
 
             <div className="block">
-              <TodoFilter />
+              <TodoFilter
+                filterBy={filterBy}
+                setFilterBy={setFilterBy}
+                query={query}
+                setQuery={setQuery}
+              />
             </div>
 
             <div className="block">
-              <Loader />
-              <TodoList />
+              {
+                isLoading
+                  ? <Loader />
+                  : todos
+                  && (
+                    <TodoList
+                      todos={filteredTodos}
+                      setIsHide={setIsHide}
+                      selectedTodo={selectedTodo as Todo}
```

Comment:
typescript warns you it can be null - please handle it correctly instead of just binding it as Todo